baseline/client/tests/n-back.test.js

```diff
@@ -1,20 +1,56 @@
 import { NBack } from "../n-back/n-back.js";
 import { pressKey, cartesianProduct } from "./utils.js"
 
+const pressBubblingKey = key => {
```

Comment:
Let's put this into test/utils.js so that it doesn't need to be repeated here and in n-back.test.js.